server/controllers/offer/getSavedOffers.js

```diff
@@ -1,3 +1,10 @@
+const { getSavedOffer } = require('../../database/queries/offers/getSavedOffer.js');
+
 module.exports = (req, res) => {
-  res.send(' get my saved offer');
+  const { memberId } = req.params;
+  getSavedOffer(memberId)
+    .then(result => res.send({
+      error: null,
+      data: result.rows,
+    }));
```

Comment:
@alaabadra It might be good to add a `catch` handler after then then, just to catch any database errors and send a 500 error back to the client.